backend/sftp/sftp.go

```diff
@@ -87,7 +87,7 @@ when the ssh-agent contains many keys.`,
 			Default: false,
 		}, {
 			Name:    "use_insecure_cipher",
-			Help:    "Enable the use of the aes128-cbc cipher and diffie-hellman-group-exchange-sha256, diffie-hellman-group-exchange-sha1 key exchange. Those algorithms are insecure and may allow plaintext data to be recovered by an attacker.",
+			Help:    "Enable the use of the aes128-cbc, aes192-cbc, aes256-cbc, and 3des-cbc ciphers and diffie-hellman-group-exchange-sha256, diffie-hellman-group-exchange-sha1 key exchange. Those algorithms are insecure and may allow plaintext data to be recovered by an attacker.",
```

Comment:
This is a bit long for a command line now!  The first line of the help is the command line

Can you split it into multiple lines something like this

```go
Help: `Enable the use of insecure ciphers and key exchange methods.

This enables the use of the the following insecure ciphers and key exchange methods:

- aes128-cbc
- aes192-cbc
- aes256-cbc
- 3des-cbc
- diffie-hellman-group-exchange-sha256
- diffie-hellman-group-exchange-sha1

Those algorithms are insecure and may allow plaintext data to be recovered by an attacker`
```

Then the command line help will be `Enable the use of insecure ciphers and key exchange methods.` and the long form will be [visible in the docs](https://tip.rclone.org/sftp/#sftp-use-insecure-cipher).

Reply:
Makes sense. I amended the help message and squashed the commits.